main.cc

```diff
@@ -676,7 +676,15 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             utils::fb_utilities::set_broadcast_rpc_address(broadcast_rpc_addr);
 
             ctx.api_dir = cfg->api_ui_dir();
+            if (!ctx.api_dir.empty() && ctx.api_dir[ctx.api_dir.length() - 1] != '/') {
```

Comment:
How about sstring::back() instead of indexing? Much shorter.

Reply:
yes, it's much nicer, I'll be pushing an updated version shortly